packages/protocol/contracts/L1/libs/LibVerifying.sol

```diff
@@ -62,7 +62,7 @@ library LibVerifying {
 
         for (
             uint256 i = state.latestVerifiedId + 1;
-            i < state.nextBlockId && processed <= maxBlocks;
+            i < state.nextBlockId && processed < maxBlocks;
```

Comment:
looks like changing `<=` to `<` is enough to fix the issue? no need to additionally pass `config.maxVerificationsPerTx + 1` as `maxBlocks`?